### PR TITLE
[Localization] #1761 Korean trainer dialogue (the other unnamed trainers)

### PR DESCRIPTION
**Pull request by sodaMelon**
[Localization] #1761 Korean trainer dialogue (other unnamed trainers)
## What are the changes?

Translating locales\ko\dialogue.ts into Korean.
This PR is added korean dialogue for the other unnamed trainers

@sodaMelon @Ohmry @returntoice now participating in https://github.com/pagefaultgames/pokerogue/issues/1761 translating project.

## Why am I doing these changes?

Few trainers's dialogue have been translated into Korean yet.

## What did change?

@malloc3141 translated **the other nonamed trainer battles' dialogues.**
@malloc3141 said you didn't have time, so I put it up instead.

malloc3141 님이 언네임드 트레이너 다이얼로그를  몇 명 더 추가로 번역해주셨어요.
디스코드에서 올릴 시간이 없다하셔서 대신 PR올립니다.

부자연스럽다면 편하게 Suggestion 주세요!

(check: 이 PR이 올라가면 unnamed trainer는 모두 번역 완료입니다.
After this PR, all the unnamed's dialogues are translated.)

### Screenshots/Videos

( very hard to meet them.... and it's dosen't matter)

## How to test the changes?

It seems there won't be much to test since it's just a translation of an existing .ts file. You can check it when you meet the respective trainers.

## Checklist
- [x] There is no overlap with another PR?
- [x] The PR is self-contained and cannot be split into smaller PRs?
- [x] Have I provided a clear explanation of the changes?
- [x] Have I tested the changes (manually)?
    - [x] Are all unit tests still passing? (`npm run test`)
- ~~[ ] Are the changes visual?~~
  - ~~[ ] Have I provided screenshots/videos of the changes?~~

### CODE REVIEW

File: src/locales/ko/dialogue.ts [hunk 1]
```diff
@@ -60,86 +60,86 @@ export const PGMdialogue: DialogueTranslationEntries = {
...
+      1: "각 포켓몬의 성격을 이해하고 육성하는 건 중요해.",
+      2: "사악한 나와는 달리 좋은 포켓몬들도 있지.",
+      3: "과도한 칭찬은 사람과 포켓몬 모두에게 독이 될 수 있어.",
     },
     "defeat": {
-      1: "You should not get angry at your Pokémon, even if you lose a battle.",
-      2: "Right? Pretty good Pokémon, huh? I'm suited to raising things.",
-      3: "No matter how much you love your Pokémon, you still have to discipline them when they misbehave."
+      1: "배틀에서 지더라도 포켓몬에게 화를 내면 안돼.",
+      2: "어때? 꽤 괜찮은 포켓몬이지? 난 뭔가를 키우는 데에 어울리는 것 같아.",
```
Comment (sodaMelon) [inline src/locales/ko/dialogue.ts:74]:
```suggestion
      2: "어때? 꽤 괜찮은 포켓몬이지? 난 무언가 기르는 게 꽤 잘 맞더라고.",
```

File: src/locales/ko/dialogue.ts [hunk 2]
```diff
@@ -60,86 +60,86 @@ export const PGMdialogue: DialogueTranslationEntries = {
...
-      3: "I have raised these very special Pokémon using a special method."
+      1: "포켓몬은 절대 널 배신하지 않아. 네가 주는 사랑을 고스란히 되돌려 준다구.",
+      2: "좋은 포켓몬을 키우는 팁을 알려줄까?",
+      3: "난 아주 특별한 방법으로 아주 특별한 포켓몬들을 키웠지!"
     },
     "victory": {
-      1: "Ugh… It wasn't supposed to be like this. Did I administer the wrong blend?",
-      2: "How could that happen to my Pokémon… What are you feeding your Pokémon?",
-      3: "If I lose, that tells you I was just killing time. It doesn't damage my ego at all."
+      1: "이런… 이렇게 될 리가 없는데. 내가 잘못된 혼합물을 썼나?",
```
Comment (sodaMelon) [inline src/locales/ko/dialogue.ts:85]:
```suggestion
      1: "이런… 이렇게 될 리가 없는데. 내가 블렌딩을 잘못 썼나?",
```
혼합물은 조금 오해의 소지가 생길 거 같아서ㅋㅋㅋ 커피나 가향차, 동물 사료 혼합에 사용하는 **블렌딩**이라는 말로 바꿨습니다.

File: src/locales/ko/dialogue.ts [hunk 3]
```diff
@@ -60,86 +60,86 @@ export const PGMdialogue: DialogueTranslationEntries = {
...
   "backpacker": {
     "encounter": {
-      1: "Pack up, game on!",
-      2: "Let's see if you can keep pace!",
-      3: "Gear up, challenger!",
-      4: "I've spent 20 years trying to find myself… But where am I?"
+      1: "게임 시작이야! 짐 단단히 싸놓으라구!",
+      2: "네가 속도를 유지할 수 있을지 보자!",
+      3: "각오 단단히 하라고!",
+      4: "20년 동안 날 찾기 위해 헤매왔었지… 그런데 내가 어디 있지?"
```
Comment (sodaMelon) [inline src/locales/ko/dialogue.ts:136]:
```suggestion
      4: "20년 동안 나 자신을 찾기 위해 헤매왔어… 근데 내가 어디 있지?"
```
구어체로 고쳐썼어요
의미는 같습니다

**Review by sodaMelon — COMMENTED** [review]
@malloc3141 잘해주셨습니다. LGTM. 
NPC스프라이트를 보면서 소리내서 읽었을 때, 어색하게 읽히는 문장들만 조금 고쳐썼어요.

아마 다른 분들이 한번 더 확인해주시면 Merge 될겁니다.

File: src/locales/ko/dialogue.ts [hunk 4]
```diff
@@ -60,86 +60,86 @@ export const PGMdialogue: DialogueTranslationEntries = {
   },
   "breeder": {
     "encounter": {
-      1: "Obedient Pokémon, selfish Pokémon… Pokémon have unique characteristics.",
-      2: "Even though my upbringing and behavior are poor, I've raised my Pokémon well.",
-      3: "Hmm, do you discipline your Pokémon? Pampering them too much is no good.",
+      1: "순종적인 포켓몬, 이기적인 포켓몬… 포켓몬들은 각기 다른 성격을 가지고 있지.",
+      2: "내 태도가 그렇게 좋진 않아도, 내 포켓몬들은 잘 키웠어.",
+      3: "음, 넌 포켓몬들을 훈련시키니? 너무 심하게 하는 것은 좋지 않다고.",
```
Comment (returntoice) [inline src/locales/ko/dialogue.ts:65]:
```suggestion
      3: "음, 넌 포켓몬들을 훈육하니? 너무 심하게 하는 것은 좋지 않다고.",
```
discipline은 훈련보다는 '훈육'이 더 어울릴 듯합니다.

**Review by returntoice — CHANGES_REQUESTED** [review]

File: src/locales/ko/dialogue.ts (hunk 4 above)
Comment (sodaMelon) [reply src/locales/ko/dialogue.ts]: 반영했습니다🤗

**Review by returntoice — APPROVED** [review]

**Comment by malloc3141** [comment]
@sodaMelon 대신 PR 올려주셔서 감사합니다 :) 수정해주신 부분도 더 자연스러워졌어요!

**Review by Tempo-anon — APPROVED** [review]

**Pull request merged**